### DISCUSSION TITLE
无法收到验证码

### DISCUSSION THREAD

**Discussion by Reynolds63272** [getlantern/lantern-forum-cn#1109] [General]
需要帮助
我2024年6月购买的lantern PRO
过去一直使用帐号、密码登录的
现在系统重装了
我下载了新版的lantern程序
![Uploading 屏幕截图 2024-12-30 125730.png…]()

但已经不能通过账号、密码登录
需要邮件收取链接码
但是我的链接码一直无法收到
请管理员协助

**Comment by Reynolds63272** [comment]
![Uploading 屏幕截图 2024-12-30 125730.png…]()

**Comment by MaxMKMax** [comment]
@Reynolds63272 您好, 请您提供我支付单据和账号邮箱. 谢谢您.

**Comment by Reynolds63272** [comment]
你好
以下截图是我的购买凭证

发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年12月31日 20:13
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: Reynolds63272 ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 无法收到验证码 (Discussion #1109)





 
@Reynolds63272 您好, 请您提供我支付单据和账号邮箱. 谢谢您.

**Comment by MaxMKMax** [comment]
@Reynolds63272 您好. 您的截图被Github屏蔽了. 请您发到我的邮箱给我. 然后也请您提供我您的账号邮箱. 谢谢您.
